Address TODO in inter-module injection. (#537)
eclipse-ee4j/cdi: spec/src/main/asciidoc/core/injectionandresolution_full.asciidoc

==== Inter-module injection in {cdi_full}

> {cdi_full} extends the set of rules defined in <<inter_module_injection>>.
> Below is a complete set of rules.

Reviewer: ```suggestion
Instead of the rules in <<inter_module_injection>>, the following rules apply in {cdi_full}.
```

Author: Hmm, I had it worded similarly originally. However, that makes it sounds like the set of rules is different when it is really just a superset. I can change it to your suggestion though, I don't mind either.

Reviewer: Or not :-) My suggestion also works :-)

Below is a complete set of rules.

A bean is also _available for injection_ in a certain module if:
A bean is _available for injection_ in a certain module if:

* the bean is not a decorator,

> * the bean is either not an alternative, or the module is a bean archive and the bean is a selected alternative of the bean archive.

Author: Oops, I meant to copy those old rules indeed but somehow copied different text. Thanks for catching this!
